webapp/channels/src/components/sidebar/sidebar_header/browse_or_add_channel_menu.tsx

```diff
@@ -0,0 +1,174 @@
+        createUserGroupMenuItem = (
+            <Menu.Item
+                id='createUserGroupMenuItem'
+                onClick={props.onCreateNewUserGroupClick}
+                leadingElement={<AccountMultiplePlusOutlineIcon size={18}/>}
+                labels={(
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.createUserGroupMenuItem.primaryLabel'
+                        defaultMessage='Create new user group'
+                    />
+                )}
+            />
+        );
+    }
+
+    let createNewCategoryMenuItem: JSX.Element | null = null;
+    if (!props.unreadFilterEnabled) {
+        createNewCategoryMenuItem = (
+            <Menu.Item
+                id='createCategoryMenuItem'
+                onClick={props.onCreateNewCategoryClick}
+                leadingElement={<FolderPlusOutlineIcon size={18}/>}
+                labels={(
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.createCategoryMenuItem.primaryLabel'
+                        defaultMessage='Create new category'
+                    />
+                )}
+            />
+        );
+    }
+
+    const invitePeopleMenuItem = (
+        <Menu.Item
+            id='invitePeopleMenuItem'
+            onClick={props.onInvitePeopleClick}
+            leadingElement={<AccountPlusOutlineIcon size={18}/>}
+            labels={(
+                <>
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.invitePeopleMenuItem.primaryLabel'
+                        defaultMessage='Invite people'
+                    />
+                    <FormattedMessage
+                        id='sidebarLeft.browserOrCreateChannelMenu.invitePeopleMenuItem.secondaryLabel'
+                        defaultMessage='Add people to the team'
+                    />
+                </>
+            )}
+            trailingElements={props.showInvitePeopleTutorialTip && <InvitePeopleTour/>}
+        />
+    );
+
+    return (
+        <Menu.Container
+            menuButton={{
+                id: ELEMENT_ID_FOR_BROWSE_OR_ADD_CHANNEL_MENU,
+                'aria-label': formatMessage({
+                    id: 'sidebarLeft.browserOrCreateChannelMenuButton.arialLabel',
+                    defaultMessage: 'Browse or create channels',
+                }),
+                class: 'btn btn-icon btn-sm btn-tertiary btn-inverted btn-round',
+                children: <PlusIcon size={18}/>,
+            }}
+            menuButtonTooltip={{
+                text: formatMessage({id: 'sidebarLeft.browserOrCreateChannelMenuButton.tooltip', defaultMessage: 'Browse or create channels'}),
+            }}
+            menu={{
+                id: 'browserOrAddChannelMenu',
+                'aria-label': formatMessage({id: 'sidebarLeft.browserOrCreateChannelMenu.ariaLabel', defaultMessage: 'Browse or create channels menu'}),
```

Comment:
I don't think this should actually include the word "menu" in it since a screen reader should read out that this is a menu already.

This is what it looks like currently in Chrome with Voiceover. Note that it reads it as "menu Browse or create channels menu"
![image](https://github.com/user-attachments/assets/701ecc3c-afcc-454f-b44d-703adb0dde9b)

Reply:
And related to my comment above about having multiple translation IDs for the same string, we can actually avoid this one entirely by using `aria-labelledby` instead of `aria-label` to name this after the button that opened the menu
```suggestion
                'aria-labelledby': ELEMENT_ID_FOR_BROWSE_OR_ADD_CHANNEL_MENU,
```

Here's the readout of how this looks when it uses `aria-labelledby`
![Screenshot 2025-01-16 at 1 13 48 PM](https://github.com/user-attachments/assets/3e3e33a7-98a0-4448-8853-3a89d187851f)
![Screenshot 2025-01-16 at 1 13 05 PM](https://github.com/user-attachments/assets/056e3beb-6d09-47e9-95be-43fa159aa632)

Reply:
done